include/vcpkg/base/messages.h

```diff
@@ -590,6 +590,15 @@ namespace vcpkg
                     "-- Automatically setting {env_var} environment variables to \"{url}\".");
     DECLARE_MESSAGE(AvailableArchitectureTriplets, (), "", "Available architecture triplets:");
     DECLARE_MESSAGE(AvailableHelpTopics, (), "", "Available help topics:");
+    DECLARE_MESSAGE(BaselineFileNoDefaultField,
+                    (msg::value),
+                    "{value} is a commit sha",
+                    "The baseline file at commit {value} was invalid (no \"default\" field).");
+    DECLARE_MESSAGE(BaselineMissingDefault,
+                    (msg::value, msg::package_name),
```

Comment:
I don't think this should be `package_name`: repo looks like a URI?

![image](https://user-images.githubusercontent.com/1544943/196821481-6d718edb-6eb1-45a2-906d-a3ec1132d0db.png)